librz/util/sdb/src/set.c

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2019 pancake <pancake@nopcode.org>
+// SPDX-License-Identifier: MIT
```

Comment:
Is `Set` API used somewhere? I don't recall any single place, probably can be removed, both **`.c`** and **`.h`** files.

Reply:
yes, for now